samples/node/ConsoleApp/README.md

```diff
@@ -0,0 +1,21 @@
+# Node Console App
+This sample demonstrates a simple console app to be run in node. 
+
+Restore dependencies
+--------------------
+```
+npm install
+dotnet restore
+```
+Build the app
+-------------------
+This will start the Fable server and invoke webpack to build and bundle the app.
+```
+dotnet fable npm-run build
```

Comment:
Probably not for this sample, trying to keep things (code, mental overhead etc.) as minimal as possible. Could use it for a bigger sample where dead code elimination has more impact